<launch>
	<node name="affw_traj" pkg="affw_traj" type="affw_traj" required="true" output="screen" args="/tmp/traj.csv" />
    	
    <param name="numIterations" value="10" />
    <param name="initPosX" value="0.5" />
    <param name="initPosY" value="-0.5" />
    <param name="initPosW" value="-0.7" />
    <param name="useInitPos" value="true" />
    	
	<arg name="export" default="true" />
	<include if="$(arg export)" file="$(find affw_export)/launch/export_all.launch" />
</launch>